libs/libsigar/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libsigar
+PKG_VERSION:=1.6.4.1
```

Comment:
Is this accurate? CMakeLists.txt says 1.6.2.

Reply:
Looks like the version string in CMakeLists hasn't been updated since 1.6.2 was released in 2009. The latest "official" version is 1.6.4 released in 2010. After that, there haven't been any releases, even in the boundary fork.
So, I just took the version 1.6.4 and append .1 at the end. I'll keep bumping the version if there are new commits that are essential.

Reply:
Typically in OpenWrt for git revisions we use PKG_SOURCE_DATE for the version. See: https://github.com/openwrt/packages/blob/master/libs/quasselc/Makefile for an example.

Having said that, this is a better way to guarantee that opkg update works properly, as long as upstream does not adopt this versioning.